wfe/wfe.go

```diff
@@ -137,6 +139,8 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h wfe
 		log: wfe.log,
 		clk: clock.Default(),
 		wfe: wfeHandlerFunc(func(ctx context.Context, logEvent *requestEvent, response http.ResponseWriter, request *http.Request) {
+			response.Header().Set("Boulder-Request-ID", logEvent.ID)
```

Comment:
This probably makes more sense in context.go right next to where the ID is created.